featuretools/tests/computational_backend/test_calculate_feature_matrix.py

```diff
@@ -1455,11 +1455,6 @@ def test_string_time_values_in_cutoff_time(es):
         calculate_feature_matrix([agg_feature], es, cutoff_time=cutoff_time)
 
 
-@pytest.fixture
-def pd_mock_customer():
-    return ft.demo.load_mock_customer(return_entityset=True, random_seed=0)
-
-
 @pytest.fixture
 def dd_mock_customer(pd_mock_customer):
     dd_mock_customer = copy.deepcopy(pd_mock_customer)
```

Comment:
We should also move the `dd_mock_customer` and `mock_customer` fixtures to `conftest.py`.